src/App.tsx

```diff
@@ -3,7 +3,7 @@ import { Box, ChakraProvider, Grid, VStack, theme } from "@chakra-ui/react";
 import {
   registerOnMessageCallback,
   registerOnSessionCallback,
-  send,
+  send,Flag1,Flag2
```

Comment:
Flag names can be descriptive - `isConnectedForLast15Seconds`

Reply:
okay